packages/config/config/devices/0x0063/templates/config_template.json

```diff
@@ -0,0 +1,468 @@
+		"valueSize": 1,
+		"defaultValue": 0,
+		"unsigned": true,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Normal operation",
+				"value": 0
+			},
+			{
+				"label": "Reset",
+				"value": 1
+			}
+		]
+	},
+	"load_sensing": {
+		"label": "Load Sensing",
+		"valueSize": 1,
+		"defaultValue": 0,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Disable",
+				"value": 0
+			},
+			{
+				"label": "Enable",
+				"value": 1
+			}
+		]
+	},
+	"switch_mode": {
+		"$import": "~/templates/master_template.json#base_enable_disable",
+		"label": "Switch Mode"
+	},
+	"report_method": {
+		"label": "Change Open/Close Report Method",
+		"valueSize": 1,
+		"defaultValue": 1,
+		"unsigned": true,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Notification",
+				"value": 1
+			},
+			{
+				"label": "Basic Set",
+				"value": 2
+			},
+			{
+				"label": "Basic report",
+				"value": 3
+			}
+		]
+	},
+	"adjust_refresh": {
+		"label": "Adjust Refresh",
+		"description": "Adjusts refresh time from 1-55 seconds",
+		"valueSize": 1,
+		"unit": "seconds",
+		"minValue": 0,
+		"maxValue": 55,
+		"defaultValue": 0,
+		"unsigned": true,
+		"options": [
+			{
+				"label": "Default - 10 seconds",
+				"value": 0
+			},
+			{
+				"label": "Seconds",
+				"value": 55
+			}
+		]
+	},
+	"alternate_exclusion": {
+		"$import": "~/templates/master_template.json#base_enable_disable",
+		"label": "Alternate Exclusion",
+		"description": "Press two times ON and two times OFF, LED will flash 5 times if successful"
+	},
+	"alternate_exclusion_h": {
+		"$import": "~/templates/master_template.json#base_enable_disable",
+		"label": "Alternate Exclusion",
+		"description": "Press 3 times and hold on the last press"
+	},
+	"alternate_exclusion_m": {
+		"$import": "~/templates/master_template.json#base_enable_disable",
+		"label": "Alternate Exclusion",
+		"description": "Press and release OFF, then ON"
+	},
+	"smooth_level_changes": {
+		"$import": "~/templates/master_template.json#base_enable_disable",
+		"label": "Smooth Level Changes"
+	},
+	"dimmer_steps": {
+		"label": "Dimmer Steps",
+		"description": "Number of dimming steps or levels",
+		"$import": "~/templates/master_template.json#base_1-99_nounit"
+	},
+	"dimmer_times": {
+		"label": "Dimmer Step Time",
+		"description": "Timing of dimming steps or levels",
+		"$import": "~/templates/master_template.json#dimming_timing"
+	},
+	"dimmer_steps_z": {
+		"label": "Dimmer Steps (Remote Command)",
+		"description": "Number of dimming steps or levels when sent from a zwave command",
+		"$import": "~/templates/master_template.json#base_1-99_nounit"
+	},
+	"dimmer_times_z": {
+		"label": "Dimmer Step Time (Remote Command)",
+		"description": "Timing of dimming steps or levels when sent from a zwave command",
+		"$import": "~/templates/master_template.json#dimming_timing"
+	},
+	"dimmer_steps_a": {
+		"label": "Dimmer Steps (Group Command)",
```

Comment:
Not sure what the difference between a Z-Wave command and a Group Command is supposed to be. Maybe the first is from the controller and the latter through associations from other nodes?